### PR TITLE
Add placeholer text to be shown if the respective recyclerView is empty

### PR DESCRIPTION
**Pull request by the-dagger**
In reference to #340 @mananwason @championswimmer 
Placeholder text has been added for Locations,LocationActivity,Tracks,Sponsors,Speakers and SpeakerActivity.
nospons
nospeak
track
cap1
capture

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java [hunk 1]
```diff
@@ -55,6 +55,8 @@
 
     private boolean flagSocial = false;
 
+    private TextView noSessions;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java]: Codacy Issue found: [Perhaps 'noSessions' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594503/issues/source?bid=3233850&fileBranchId=3233896#l58)

File: app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java [hunk 2]
```diff
@@ -55,6 +55,8 @@
...
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java]: Codacy Issue found: [Avoid really long methods.](https://www.codacy.com/app/triplez/open-event-android/file/2406594503/issues/source?bid=3233850&fileBranchId=3233896#l61)

File: app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java [hunk 3]
```diff
@@ -42,6 +43,8 @@
 
     private SwipeRefreshLayout swipeRefreshLayout;
 
+    private TextView noSponsors;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java]: Codacy Issue found: [Perhaps 'noSponsors' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594560/issues/source?bid=3233850&fileBranchId=3233896#l46)

File: app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java [hunk 4]
```diff
@@ -46,22 +47,33 @@
 
     private SearchView searchView;
 
+    private TextView noLocationSessions;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java]: Codacy Issue found: [Perhaps 'noLocationSessions' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594499/issues/source?bid=3233850&fileBranchId=3233896#l50)

File: app/src/main/java/org/fossasia/openevent/fragments/SpeakerFragment.java [hunk 5]
```diff
@@ -71,16 +72,27 @@
 
     private int sortType;
 
+    private TextView noSpeakers;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/SpeakerFragment.java]: Codacy Issue found: [Perhaps 'noSpeakers' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594559/issues/source?bid=3233850&fileBranchId=3233896#l75)

File: app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java [hunk 6]
```diff
@@ -47,15 +48,27 @@
 
     private SearchView searchView;
 
+    private TextView noLocations;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java]: Codacy Issue found: [Perhaps 'noLocations' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594558/issues/source?bid=3233850&fileBranchId=3233896#l51)

File: app/src/main/java/org/fossasia/openevent/fragments/TracksFragment.java [hunk 7]
```diff
@@ -54,16 +55,24 @@
     private String searchText = "";
 
     private SearchView searchView;
-
+    private TextView noTracks;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/TracksFragment.java]: Codacy Issue found: [Perhaps 'noTracks' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594561/issues/source?bid=3233850&fileBranchId=3233896#l58)

File: app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java (hunk 1 above)
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java]: Codacy Issue found: [Perhaps 'noSessions' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594503/issues/source?bid=3233850&fileBranchId=3233896#l58)

File: app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java (hunk 2 above)
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java]: Codacy Issue found: [Avoid really long methods.](https://www.codacy.com/app/triplez/open-event-android/file/2406594503/issues/source?bid=3233850&fileBranchId=3233896#l61)

File: app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java (hunk 3 above)
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java]: Codacy Issue found: [Perhaps 'noSponsors' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594560/issues/source?bid=3233850&fileBranchId=3233896#l46)

File: app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java (hunk 4 above)
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java]: Codacy Issue found: [Perhaps 'noLocationSessions' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594499/issues/source?bid=3233850&fileBranchId=3233896#l50)

File: app/src/main/java/org/fossasia/openevent/fragments/SpeakerFragment.java (hunk 5 above)
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/SpeakerFragment.java]: Codacy Issue found: [Perhaps 'noSpeakers' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594559/issues/source?bid=3233850&fileBranchId=3233896#l75)

File: app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java (hunk 6 above)
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java]: Codacy Issue found: [Perhaps 'noLocations' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594558/issues/source?bid=3233850&fileBranchId=3233896#l51)

File: app/src/main/java/org/fossasia/openevent/fragments/TracksFragment.java (hunk 7 above)
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/TracksFragment.java]: Codacy Issue found: [Perhaps 'noTracks' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594561/issues/source?bid=3233850&fileBranchId=3233896#l58)

File: app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java (hunk 6 above)
Comment (the-dagger) [inline app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java]: Will do.

**Pull request closed without merging**